### DISCUSSION TITLE
Laravel Horizon with multiple apps

### DISCUSSION THREAD

**Discussion by cay89** [laravel/framework#49390] [Q&A] [answered]
I couldn't find an exact answer for this on the web. Can I use a standalone installed Laravel Horizon instance to display the queue content of multiple Laravel apps? 

![image](https://github.com/laravel/framework/assets/3186515/8aafe3db-f196-4c81-983e-920fba342213)

Or is it designed for a 'one horizon per app' use case?

**Comment by rdgout** [comment, marked as answer]
Horizon runs on your app and manages the job queues. The config of horizon knows where to look for queues and jobs because the application is configured a certain way.
What you want to do is probably possible in some limited way but it probably requires a whole lot of effort.
